src/walletConnect/screens/ActionRequest.tsx

```diff
@@ -15,13 +19,22 @@ import RequestContent, { useDappMetadata } from 'src/walletConnect/screens/Reque
 import { useIsDappListed } from 'src/walletConnect/screens/useIsDappListed'
 import { sessionsSelector } from 'src/walletConnect/selectors'
 
-interface Props {
+export interface ActionRequestProps {
   version: 2
   pendingAction: Web3WalletTypes.EventArguments['session_request']
   supportedChains: string[]
+  hasInsufficientGasFunds: boolean
+  feeCurrencies: TokenBalance[]
+  preparedTransaction?: SerializableTransactionRequest
 }
 
-function ActionRequest({ pendingAction, supportedChains }: Props) {
+function ActionRequest({
+  pendingAction,
+  supportedChains,
+  hasInsufficientGasFunds,
+  feeCurrencies,
+  preparedTransaction,
```

Comment:
we still display the initial unaltered transaction in the bottom sheet. will we address it in another task?

Reply:
thanks for the catch! i wasn't thinking about this, seems to make sense to add it here so i've updated the corresponding component